staging/jenkins/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
I think it's important to note that `staging/` is meant to be used for charts that are going to be worked on here and then PR'd against upstream implementations. I just wanted to double check: is that the goal here?

Reply:
https://github.com/mesosphere/charts#repository-structure
staging is for temporary forks where there is a pending PR upstream.